### PR TITLE
adjust help text for easier RPi3, BBB and Yocto use with Hosted Mender

### PR DESCRIPTION
**Pull request by estenberg**
Signed-off-by: Eystein Måløy Stenberg <eystein.maloy.stenberg@cfengine.com>

### CODE REVIEW

File: src/js/components/help/connecting-devices/beagleboneblack.js [hunk 1]
```diff
@@ -89,10 +90,25 @@ var BeagleBoneBlack =  createReactClass({
 
         <p> If you cannot see any new device after 10 minutes, please verify that the network connection is working, and feel free to reach out for help with diagnostics via <a onClick={this.props.changePage.bind(null, "help/more-help-resources")}>our further Help resources</a>.</p>
 
+        <p> NOTE: If you reimage the device with the disk image or switch storage (e.g. SD cards) between the devices after they have booted once, <b>authenticating the device to the Mender server will fail</b>. This is because the Mender server tracks the identity (MAC address by default) <-> device public key (randomly generated upon first run of the Mender client) binding. If this happens, you need to *decommission* the device from the Mender server and try again. <p>
+
+
+        <h3>Deploy updates</h3>
+
+        {this.props.isHosted ? 
+          <p>Download these two BeagleBone Black Artifacts customized for your Hosted Mender account: TODO LINKS: Artifact1, [Artifact2].</p>
```
Comment (estenberg) [inline src/js/components/help/connecting-devices/beagleboneblack.js]: Needs adjustment for BBB Artifact 1 & BBB Artifact 2 URLs.

File: src/js/components/help/connecting-devices/raspberry-pi-3.js [hunk 2]
```diff
@@ -85,11 +87,27 @@ var RaspberryPi =  createReactClass({
...
+        <p> If you cannot see any new device after 10 minutes, please verify that the network connection is working, and feel free to reach out for help with diagnostics via <a onClick={this.props.changePage.bind(null, "help/more-help-resources")}>our further Help resources</a>.</p>
+
+        <p> NOTE: If you reimage the device with the disk image or switch storage (e.g. SD cards) between the devices after they have booted once, <b>authenticating the device to the Mender server will fail</b>. This is because the Mender server tracks the identity (MAC address by default) <-> device public key (randomly generated upon first run of the Mender client) binding. If this happens, you need to *decommission* the device from the Mender server and try again. <p>
+
+
+        <h3>Deploy updates</h3>
+
+        {this.props.isHosted ? 
+          <p>Download these two Raspberry Pi 3 Artifacts customized for your Hosted Mender account: TODO LINKS: Artifact1, [Artifact2].</p>
```
Comment (estenberg) [inline src/js/components/help/connecting-devices/raspberry-pi-3.js]: Needs adjustment for Rpi3 Artifact 1 & Rpi3 Artifact 2 URLs.

Comment (michaelatmender) [reply src/js/components/help/connecting-devices/raspberry-pi-3.js]: @estenberg: The `<->` after "(MAC address by default)" causes an issue when running gulp. Is it a typo?

**Comment by michaelatmender** [comment]
I made a new PR with these changes including a few fixes and the image links: #317

**Pull request closed without merging**